Assets/Player.unity

```diff
@@ -182,8 +186,33 @@ PrefabInstance:
       propertyPath: m_Enabled
       value: 1
       objectReference: {fileID: 0}
-    m_RemovedComponents: []
+    m_RemovedComponents:
+    - {fileID: 8532516662099360271, guid: 52d8a1efc00b9e046a087f0d9748cf9f, type: 3}
   m_SourcePrefab: {fileID: 100100000, guid: 52d8a1efc00b9e046a087f0d9748cf9f, type: 3}
+--- !u!1 &898726581 stripped
```

Comment:
Há mudanças no Player.unity.
Recomendação: Aplicar todas as mudanças ao prefab com o botão "override".

Reply:
Se não for possível aplicar todas, tudo bem. Creio que não será pela criação da nova tag, Mas de boa, essa semana n tem ninguém mexendo no player.unity, então creio que não gerará conflitos sérios.

Reply:
Essas mudanças iniciais devem ter sido porque eu movi o object_spawner para ver se ele estava centralizado. Mas não fiz grandes alterações que não possam ser excluídas. 

Na verdade eu não criei nenhuma tag, só adicionei a que já existia ao objeto (ele estava sem nenhuma e por isso não conseguia ser identificado usando a busca por tag). 

Talvez as alterações no player.unity possam ter acontecido também porque o script gera obstáculos na cena, o que teoricamente causa uma alteração...

Reply:
Creio que os objetos spawnados não sejam a razão, pois os objetos são criados durante runtime. Isso não altera nada em memória não volátil. 
E tudo relacionado ao spawn está dentro de um prefab "game_manager", então se vc aplicar os ovewrites deve concertar tudo.

Reply:
![image](https://user-images.githubusercontent.com/39244876/107696548-dcc63600-6c90-11eb-8b56-ebfe0790854d.png)

Reply:
Show, vou verificar isso!